From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pavel Tumakaev <p.tumakaev@omprussia.ru>
Date: Mon, 15 Feb 2021 19:11:51 +0300
Subject: [PATCH 11/67] Disable MOC code generation for message_pump_qt

This patch removes signals/slots from MessagePumpQt and disables MOC code
generation for message_pump_qt.

Signed-off-by: Pavel Tumakaev <p.tumakaev@omprussia.ru>
---
 ipc/chromium/moz.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipc/chromium/moz.build b/ipc/chromium/moz.build
index b808b21296a5..6bd3b11d2351 100644
--- a/ipc/chromium/moz.build
+++ b/ipc/chromium/moz.build
@@ -119,9 +119,9 @@ if os_bsd or os_linux:
         ]
     if CONFIG['MOZ_ENABLE_QT']:
         SOURCES += [
-            '!moc_message_pump_qt.cc',
             'src/base/message_pump_qt.cc',
         ]
+        CXXFLAGS += CONFIG['MOZ_QT_CFLAGS']
 
 if os_solaris:
     SOURCES += [
